pulp_container/app/registry_api.py

```diff
@@ -234,6 +259,21 @@ def __init__(self, manifest, path, request, status=200):
         super().__init__(headers=headers, status=status, content_type=manifest.media_type)
 
 
+class ManifestSignatureResponse(Response):
+    """
+    An HTTP response class after creating an image signature.
+    """
+
+    def __init__(self, signature, path, status=201):
+        """Initialize the headers with the path to the repository and corresponding digests."""
+        headers = {
+            "Docker-Content-Digest": signature.digest,
```

Comment:
it cannot be signature.digest. PUT can contain a list of signature for 1 image, if anything then it should be digest of the image but in case of docker-content-digest header - usually this header contains digest of the object that has been just created/fetched.
tldr; remove this header